src/main/java/frc/robot/commands/GetAlgae.java

```diff
@@ -48,6 +51,11 @@ public GetAlgae(DriveSubsystem drive, HandlerSubsystem handler, Crane crane,
       Commands.waitUntil(() -> m_crane.atGoal().isPresent()),
 
       Commands.runOnce(() -> m_handler.intakeAlgae()),
+      Commands.defer((() -> {
+        Transform2d move = new Transform2d(AutoConstants.kAlgaeIntakePositionTwo, 0.0, new Rotation2d());
```

Comment:
`move` can be reused. In fact, we could make `kAlgaeIntakePositionTwo` a `Transforrm2d` (maybe as `kAlgaeIntakeMove`).